docs/tutorial.rst

```diff
@@ -282,7 +293,7 @@ using SLiM with a (very large!) scaling factor of 1000, we could run
 
 .. code-block:: console
 
-   $ stdpopsim -e slim --slim-scaling-factor 1000 DroMel \
```

Comment:
Does this work? I thought ``--engine`` options were to the top level parser and not available to the subcommands.

Reply:
Ah, you're right. It guess it looked weird to have the species "positional" argument coming in the middle of some optional flags. In general, the CLI interface is confusing and doesn't follow standard guidelines, but that's another matter. I'll change it back so it works with what we've got.